angular-viewport-watch.js

```diff
@@ -28,6 +28,14 @@
                     return unwatch;
                 }
                 function toggleWatchers(scope, enable) {
+                    // After the addition of sticky states, we want to ignore toggling the
+                    // watchers when these elements are not being displayed.
+                    if (attr.viewportWatch && !$state.includes(attr.viewportWatch)) {
```

Comment:
If the element is not in the current state should we still turn the watches off?